web-app/src/main/java/com/google/sps/data/QuadTree.java

```diff
@@ -3,18 +3,17 @@
 import java.util.ArrayDeque;
 import java.util.ArrayList;
 import java.util.Deque;
-import com.google.gson.Gson;
-import com.google.sps.data.PoliceReport;
-import com.google.sps.data.Rectangle;
 
 public class QuadTree {
-  private Node root;
+  public Node root;
```

Comment:
I wouldn't make this `public`. It gives too much control for users of this API. They can break your classes invariants (e.g. create a Node with more reports than capacity).